include/spdlog/details/async_log_helper.h

```diff
@@ -88,8 +88,8 @@ class async_log_helper
             msg.level = level;
             msg.time = time;
             msg.thread_id = thread_id;
-            msg.raw.clear();
-            msg.raw << txt;
+            msg.raw = fmt::memory_buffer();
```

Comment:
I don't think this will be an issue, but `memory_buffer` doesn't seem to include any destructive methods like `.clear` aside from this operator. I believe this will result in a clean deallocation and essentially nothing else.